package.json

```diff
@@ -38,6 +38,7 @@
     "@material-ui/lab": "^4.0.0-alpha.61",
     "@material-ui/styles": "^4.11.4",
     "@radix-ui/react-portal": "^1.1.4",
+    "lucide-react": "^0.534.0",
```

Comment:
omg yes ❤️

Reply:
Let's instantly deprecate all other icons

Reply:
Yes - to be tracked in https://linear.app/saleor/issue/ENG-447